lib/wallet/rpc.js

```diff
@@ -752,7 +752,7 @@ class RPC extends RPCBase {
     return {
       walletid: wallet.id,
       walletversion: 6,
-      balance: Amount.coin(balance.unconfirmed, true),
+      balance: Amount.coin(balance.confirmed, true),
```

Comment:
Ok my thinking on this:  Let's not break backwards compatibility by changing what is returned here. We don't want to mess up any application relying on this value always being the "safest" (confirmed) balance.

What I think could be useful is just adding more values. We already have `unconfirmed_balance`, let's just add `confirmed_balance` and `spendable_balance` (combine with https://github.com/handshake-org/hsd/pull/507)